src/shared/sleep-config.h

```diff
@@ -5,7 +5,7 @@
 #include "time-util.h"
 
 int read_fiemap(int fd, struct fiemap **ret);
-int parse_sleep_config(const char *verb, char ***modes, char ***states, usec_t *delay);
+int parse_sleep_config(const char *verb, bool *_allow, char ***_modes, char ***_states, usec_t *_delay);
```

Comment:
i think it would be nice to prefix return parameters with `ret_`, we have been doing that at many calls, and in this case that would make sense too I think.